assets/javascripts/discourse-assign/initializers/extend-for-assigns.js.es6

```diff
@@ -506,6 +506,24 @@ function initialize(api) {
     },
   });
 
+  api.modifyClass("component:topic-notifications-button", {
+    @discourseComputed(
+      "topic",
+      "topic.details.{notification_level,notifications_reason_id}"
+    )
+    notificationReasonText(topic) {
+      if (
+        this.currentUser.auto_track_topics_after_msecs === -1 &&
```

Comment:
The `-1` is sort of like a magic number. I'm wondering if we can move this check to the server side where we ship down a boolean field `auto_track_disabled` instead. 

```
  add_to_serializer(:current_user, :auto_track_disabled) do
    user.user_option.auto_track_topics_after_msecs == AutoTrackDurationSiteSetting.values[:never]
  end
```

^^ The above is a sample of what I'm thinking about and the code might not work.

Reply:
I could add a `auto_track_disabled` into the serializer instead of this setting.